kirin/piv/model_maker.py

```diff
@@ -266,17 +306,25 @@ def _get_navitia_vj(self, piv_key, ads):
             raise ObjectNotFound("no train found for key {}".format(piv_key))
         return vj
 
-    def _make_trip_update(self, json_train, vj):
+    def _make_trip_update(self, json_train, vj, trip_status):
         trip_update = model.TripUpdate(vj=vj, contributor_id=self.contributor.id)
         trip_update.headsign = get_value(json_train, "numero", nullable=True)
         company_id = get_value(get_value(json_train, "operateur"), "codeOperateur")
         trip_update.company_id = self._get_navitia_company(company_id)
         physical_mode = get_value(get_value(json_train, "modeTransport"), "typeMode")
         trip_update.physical_mode_id = self._get_navitia_physical_mode(physical_mode)
+        trip_status_type = trip_status[0]
+        trip_update.message = trip_status[1]
+        trip_update.effect = trip_status_type.name
+
+        # TODO: handle status/effect for creation, detour, back-to-normal
+        if trip_status_type == TripEffect.NO_SERVICE:  # SUPPRESSION or SUPPRESSION_TOTALE
```

Comment:
`SUPPRESSION_TOTALE` is not a possible value anymore it seems, and all the comment can be removed I think:
```suggestion
        if trip_status_type == TripEffect.NO_SERVICE:
```